src/argilla/client/feedback/schemas.py

```diff
@@ -183,12 +183,18 @@ class FeedbackRecord(BaseModel):
     fields: Dict[str, str]
     metadata: Dict[str, Any] = Field(default_factory=dict)
     responses: List[ResponseSchema] = Field(default_factory=list)
-    suggestions: List[SuggestionSchema] = Field(default_factory=list, allow_mutation=True)
+    suggestions: Union[Tuple[SuggestionSchema], List[SuggestionSchema]] = Field(
+        default_factory=tuple, allow_mutation=False
+    )
     external_id: Optional[str] = None
 
     _unified_responses: Optional[Dict[str, List["UnifiedValueSchema"]]] = PrivateAttr(default_factory=dict)
     _updated: bool = PrivateAttr(default=False)
 
+    @validator("suggestions", always=True)
+    def normalize_suggestions(cls, values) -> Tuple:
+        return tuple([v for v in values])
```

Comment:
Here we are missing `values` type-hint and the `return` just applies if not a `Tuple` already right? So shouldn't it be something like
```suggestion
    def normalize_suggestions(cls, values: Any) -> Tuple:
        if not isinstance(values, tuple):
            return tuple([v for v in values])
        return values
```